.github/workflows/main_push.yml

```diff
@@ -60,46 +54,20 @@ jobs:
 
       - uses: Alfresco/alfresco-build-tools/.github/actions/update-pom-to-next-pre-release@f2259c2bb5ff4a5579d1a425cb130175a6459665 # v5.10.0
```

Comment:
The step: `Check dependabot build` can be removed as this is targeting push only

Reply:
https://github.com/Activiti/Activiti/pull/4529/commits/d5f96261875a6604378c59939a4d890b91061612